### DISCUSSION TITLE
Unable to Install OMZ

### DISCUSSION THREAD

**Discussion by ryanburnett** [ohmyzsh/ohmyzsh#9967] [Help] [answered]
### How to reproduce the problem
`sh -c "$(curl -fsSL https://raw.github.com/ohmyzsh/ohmyzsh/master/tools/install.sh)"`

<img width="682" alt="screen shot 2021-06-12 at 11 35 47 pm" src="https://user-images.githubusercontent.com/741858/121794690-70c0f380-cbd8-11eb-9ea1-c252df741e10.png">

`echo $SHELL` (returns /bin/zsh)

<img width="682" alt="screen shot 2021-06-12 at 11 38 01 pm" src="https://user-images.githubusercontent.com/741858/121794709-8fbf8580-cbd8-11eb-9a81-897e565f0fe2.png">

Command + Q (asks to terminate running processes)

<img width="682" alt="screen shot 2021-06-12 at 11 38 08 pm" src="https://user-images.githubusercontent.com/741858/121794719-a4038280-cbd8-11eb-9d7a-806f11c164e5.png">

### What the correct behavior should be

1. Oh My Zsh should install successfully and ask if I want to change my shell to Oh My Zsh
2. `echo $SHELL` should return Oh My Zsh (instead of /bin/zsh)
3. I should be able to quit the Terminal without having to terminate any processes

### What the actual behavior is

The installation is hanging up on something and not changing my shell to Oh My Zsh.

**Comment by mcornella** [comment, marked as answer]
That's because the changes haven't been applied yet. Also, when you install OMZ it'll run `zsh` for you, but that's still on the child process of the previous shell. Since the previous shell is still running that's why you get the Terminal prompt. We can't do anything about this.

Just close the shell and reopen it, and you'll be good to go.
